src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java

```diff
@@ -229,14 +230,27 @@ private static String combineAddressLines(String... lines) {
         return joiner.toString();
     }
 
-    public static boolean equalsIndividualNationality(String string, String other) {
-        var nullValuesCheck = handleNulls(string, other);
+    public static boolean equalsIndividualNationality(String str, String other) {
+        var nullValuesCheck = handleNulls(str, other);
         if (nullValuesCheck != null) {
             return nullValuesCheck;
         }
-        String normalisedNationality =
-                other.endsWith(",") ? other.substring(0, other.length() - 1) : other;
-        return StringUtils.equalsIgnoreCase(normalise(string), normalise(normalisedNationality));
+
+        var normalisedStr = normaliseCommaSeparatedList(str);
+        var normalisedOther = normaliseCommaSeparatedList(other);
+
+        return StringUtils.equalsIgnoreCase(normalisedStr, normalisedOther);
+    }
+
+    private static String normaliseCommaSeparatedList(String list){
+        if(list.contains(",")){
+            return Arrays.stream(list.split(","))
+                    .map(ComparisonHelper::normalise)
+                    .filter(Objects::nonNull)
```

Comment:
filter(StringUtils::isNotBlank)

Reply:
Normalise already does isNotBlank and returns a null if it comes back true.

If we did `StringUtils::isNotBlank` we'd just be running the same code twice.

Reply:
Well actually it uses `StringUtils::normalizeSpace` then `String::isBlank` but that has the same effect as `StringUtils::isNotBlank`

Reply:
You use isNotBlank below. I now think normalise (the map func)  will never produce a null and only "".

Reply:
so we don't need the filter

Reply:
Normalise returns null if `text` the result of `text.isBlank()` is true
![image](https://github.com/companieshouse/overseas-entities-api/assets/124157110/cf037117-404b-4a6d-bbe4-5b8713057aae)

This means that without the filter something like "a, " will become "a,null"


_Looking at that normalise again we should have perhaps done `StringUtils.isBlank()` instead of the null check but that's a tangent_